app_notifier/src/app_notifier/mail_notifier_plugin.py

```diff
@@ -27,8 +37,12 @@ def app_manager_stop_plugin(self, app, ctx, plugin_args):
             use_timestamp_title = plugin_args['use_timestamp_title']
 
         if use_timestamp_title:
-            timestamp = '{0:%Y/%m/%d (%H:%M:%S)}'.format(
-                datetime.datetime.now())
+            if self.use_app_start_time:
+                timestamp = '{0:%Y/%m/%d (%H:%M:%S)}'.format(
+                    datetime.datetime.fromtimestamp(self.start_time.to_sec()))
+            else:
+                timestamp = '{0:%Y/%m/%d (%H:%M:%S)}'.format(
+                    datetime.datetime.now())
```

Comment:
`'{0:%Y/%m/%d (%H:%M:%S)}'.format(` is used for several times.
You can create method to convert ROS time to timestamp.
Or
```
            if self.use_app_start_time:
                unix_time = self.start_time.to_sec()
            else:
                unix_time = datetime.datetime.now()
            timestamp = '{0:%Y/%m/%d (%H:%M:%S)}'.format(
                datetime.datetime.fromtimestamp(unix))
```

Reply:
Thank you very much for advice.
I used latter one [216eade](https://github.com/knorth55/app_manager_utils/pull/55/commits/216eadeccd48dea7f4b1ab2c34db5ada3c09267c)